website/alicloud.erb

```diff
@@ -554,6 +554,22 @@
                         </li>
                     </ul>
                 </li>
+
+                <li<%= sidebar_current("docs-alicloud-resource-datahub-project") %>>
```

Comment:
I think line 558 should be ` <li<%= sidebar_current("docs-alicloud-resource-datahub") %>>`